tests/act.rs

```diff
@@ -1,119 +1,143 @@
     assert!(lhs == rhs);
     assert!(rhs == lhs);
 }
 
-#[test]
-fn ord_eq_if_contains_key_in_lhs(){
-    let key = Act::KeyChord(Modifiers::CONTROL, KeyCode::A);
-    let lhs = ActPattern::Any(HashSet::from_iter([key.clone()]));
-    let rhs = ActPattern::One(key);
-    assert_eq!(lhs.cmp(&rhs), Ordering::Equal);
-}
-
-#[test]
-fn ord_not_eq_if_non_contains_key_in_lhs(){
-    let a = Act::KeyChord(Modifiers::CONTROL, KeyCode::A);
-    let b = Act::KeyChord(Modifiers::CONTROL, KeyCode::B);
-    let lhs = ActPattern::Any(HashSet::from_iter([a]));
-    let rhs = ActPattern::One(b);
-    // If not Equal, `lhs` is Greater or Less? It needs to be implemented with consistency.
-    assert_eq!(lhs.cmp(&rhs), Ordering::Less);
-}
+// #[test]
+// fn test_shifted_key_macro() {
+//     assert_eq!((Modifiers::CONTROL, KeyCode::KeyB), key! { ctrl-* });
+// }
 
 /// XXX: This doc test isn't working.
 ///
 /// ```compile_fail
 /// assert_eq!((Modifiers::CONTROL, KeyCode::F2), key!{ ctrl-f2 });
 /// ```
+///
+/// ```
+/// let _ = key! { ctrl-* });
+/// ```
 #[allow(unused_must_use)]
 #[test]
 fn test_key_macro() {
-    assert_eq!((Modifiers::CONTROL, KeyCode::B), key! { ctrl-B });
-    assert_eq!((Modifiers::CONTROL, KeyCode::Key1), key! { ctrl-1 });
-    assert_eq!((Modifiers::CONTROL, KeyCode::Key2), key! { ctrl-2 });
+    assert_eq!((Modifiers::CONTROL, KeyCode::KeyB), key! { ctrl-B });
+    assert_eq!((Modifiers::CONTROL, KeyCode::Digit1), key! { ctrl-1 });
+    assert_eq!((Modifiers::CONTROL, KeyCode::Digit2), key! { ctrl-2 });
     assert_eq!((Modifiers::CONTROL, KeyCode::F2), key! { ctrl-F2 });
     // assert_eq!((Modifiers::CONTROL, KeyCode::F2), key!{ ctrl-f2 });
     assert_eq!((Modifiers::CONTROL, KeyCode::Semicolon), key! { ctrl-; });
-    assert_eq!((Modifiers::CONTROL, KeyCode::Caret), key! { ctrl-^ });
+    // assert_eq!((Modifiers::CONTROL, KeyCode::Caret), key! { ctrl-^ });
     // assert_eq!((Modifiers::CONTROL, KeyCode::Colon), key! { ctrl-: });
-    assert_eq!((Modifiers::CONTROL, KeyCode::Equals), key! { ctrl-= });
+    assert_eq!((Modifiers::CONTROL, KeyCode::Equal), key! { ctrl-= });
     assert_eq!((Modifiers::CONTROL, KeyCode::Comma), key! { ctrl-, });
     assert_eq!((Modifiers::CONTROL, KeyCode::Period), key! { ctrl-. });
     assert_eq!((Modifiers::CONTROL, KeyCode::Slash), key! { ctrl-/ });
+    assert_eq!((Modifiers::CONTROL, KeyCode::Enter), key! { ctrl-Enter });
+    assert_eq!((Modifiers::CONTROL, KeyCode::Space), key! { ctrl-Space });
+    assert_eq!((Modifiers::CONTROL, KeyCode::Tab), key! { ctrl-Tab });
+    assert_eq!((Modifiers::CONTROL, KeyCode::Delete), key! { ctrl-Delete });
     assert_eq!((Modifiers::CONTROL, KeyCode::Minus), key! { ctrl-- });
-    assert_eq!((Modifiers::CONTROL, KeyCode::Underline), key! { ctrl-_ });
-    assert_eq!((Modifiers::CONTROL, KeyCode::Colon), key! { ctrl-: });
     assert_eq!(
-        (Modifiers::CONTROL, KeyCode::Apostrophe),
-        key! { ctrl-'\'' }
```

Comment:
`cargo test` was failed; see below for the error log.

```
error[E0599]: no variant or associated item named `Grave` found for enum `bevy::prelude::KeyCode` in the current scope
   --> tests\act.rs:125:58
    |
125 |     assert_eq!((Modifiers::CONTROL, KeyCode::Backquote), key! { ctrl-'`' });
    |                                                          ^^^^^^^^^^^^^^^^^ variant or associated item not found in `KeyCode` 
```

Reply:
Doh! There are commits to keyseq on my desktop computer that I need to push and publish. Unfortunately I'm going to be away from my desktop computer until Thursday. :( Apologies. I'll comment here when I get that sorted out.

Reply:
Yeah, Best regards!